cms/djangoapps/contentstore/management/commands/reindex_course.py

```diff
@@ -62,6 +62,40 @@ def _parse_course_key(self, raw_value):
 
         return result
 
+    def _setup(self):
```

Comment:
this could be named something more accurate, it is checking something about the indices I can't quite understand but it's definitely not setup

Reply:
Oh I see it is checking "are the indices ALREADY set up" so it is setup except that we always use setup all over the code as an action not a question

mabye "indexes_already_setup" or "indices_exist" or "indices_need_setup"

Reply:
Refactored it.